### PR TITLE
Add L3_HA as feature for neutron routers

### PR DESCRIPTION
**Pull request by ssrikant20**

### CODE REVIEW

**Comment by dlundquist** [comment]
ok to test

**Comment by dlundquist** [comment]
Looks good!

File: roles/neutron-common/templates/etc/neutron/rootwrap.conf [hunk 1]
```diff
@@ -12,9 +12,9 @@ filters_path=/etc/neutron/rootwrap.d,/usr/share/neutron/rootwrap,/etc/neutron/ro
 # If not specified, defaults to system PATH environment variable.
 # These directories MUST all be only writeable by root !
 {% if openstack_install_method == "package" -%}
-exec_dirs=/sbin,/usr/sbin,/bin,/usr/bin,{{ 'neutron'|ursula_package_path(openstack_package_version) }}/bin
+exec_dirs=/sbin,/usr/sbin,/bin,/usr/bin,/usr/local/bin,{{ 'neutron'|ursula_package_path(openstack_package_version) }}/bin
 {% else -%}
-exec_dirs=/sbin,/usr/sbin,/bin,/usr/bin
+exec_dirs=/sbin,/usr/sbin,/bin,/usr/bin,/usr/local/bin
```
Comment (j2sol) [inline roles/neutron-common/templates/etc/neutron/rootwrap.conf]: This is going to break, because of source installs being in venvs as well. I thought I had provided a sample template for this in one of the other PRs?

Comment (ssrikant20) [inline roles/neutron-common/templates/etc/neutron/rootwrap.conf]: @j2sol: In the last pr, you had mentioned that we copy all binaries to /usr/local/bin and path may not be needed and to use /usr/local/bin for now.
https://github.com/blueboxgroup/ursula/pull/1310

Comment (ssrikant20) [inline roles/neutron-common/templates/etc/neutron/rootwrap.conf]: Do I need to patch 
exec_dirs=/sbin,/usr/sbin,/bin,/usr/bin,/usr/local/bin,{{ openstac-source.virtualenv_base }}/neutron/bin

Comment (j2sol) [inline roles/neutron-common/templates/etc/neutron/rootwrap.conf]: Oh, we solved this in another place didn't we. You're right, what we should do here is take the package path addition out of exec_dirs, since we copy all the bins into /usr/local/bin, and we shouldn't be execing anything directly in the venv path. Can you do that and confirm things still work?

Comment (j2sol) [inline roles/neutron-common/templates/etc/neutron/rootwrap.conf]: Leave /usr/local/bin, take out the templated openstack-source stuff, and take out the { 'neutron'|ursula_package_path(openstack_package_version) }} bits from the exec_dirs for package based install. In fact, that whole if statement can go away, it should be just a single set of paths.

Comment (ssrikant20) [inline roles/neutron-common/templates/etc/neutron/rootwrap.conf]: Sure.. Should that be part of this PR? I will make those changes separately and we will test it in our test env and make a new pr. Let me know.

Comment (j2sol) [inline roles/neutron-common/templates/etc/neutron/rootwrap.conf]: Lets do it as a separate PR, and leave this bit of the file as is for this PR.

Comment (ssrikant20) [inline roles/neutron-common/templates/etc/neutron/rootwrap.conf]: Thanks. I am converging my test env with the suggested change and we will do some testing. Once we get results, will make a new pr for it.
